site/public/css/submitbox.css

```diff
@@ -35,6 +35,26 @@
     height: 150px;
 }
 
+.fileUploadTable {
+    border: none;
+    border-collapse: collapse;
+    text-align: left;
+    margin: 40px auto;
+}
+
+.fileUploadTable td {
+    border: none;
+    padding: 15px 20px;
+}
+
+.fileUploadTable td:nth-child(even){
+    text-align: right;
+}
+  
+.fileUploadTable tr:hover {
+    background: #d1d1d1;
+}
+
```

Comment:
The names of classes (and ids) should be in `kebab-case` (e.g. `file-upload-table`).